go.mod

```diff
@@ -237,7 +237,7 @@ replace (
 
 	github.com/peggyjv/gravity-bridge/module/v2 => github.com/crypto-org-chain/gravity-bridge/module/v2 v2.0.1-0.20230126051749-d984b1562242
 	// use cometbft
-	github.com/tendermint/tendermint => github.com/cometbft/cometbft v0.34.27
+	github.com/tendermint/tendermint => github.com/cometbft/cometbft v0.34.28
```

Comment:
maybe use v0.34.28 for submodules as well

Reply:
submodules can wait for a point release of cosmos-sdk I think.

Reply:
seems has behaviour change in statesync https://github.com/cometbft/cometbft/compare/v0.34.27...v0.34.28

Reply:
yeah, statesync test case failed after upgrade, but I haven't find out where to blame.

Reply:
seems can pass when revert https://github.com/crypto-org-chain/ethermint/pull/242/files

Reply:
yeah, goleveldb is the key 👍

Reply:
https://github.com/cosmos/cosmos-sdk/pull/14952